xinference/model/llm/llm_family.py

```diff
@@ -447,6 +447,61 @@ def cache_from_uri(
         raise ValueError(f"Unsupported URL scheme: {src_scheme}")
 
 
+def cache_model_config(
+    llm_family: LLMFamilyV1,
+    llm_spec: "LLMSpecV1",
+):
+    """Download model config.json into cache_dir,
+    returns local filepath
+    """
+    cache_dir = _get_cache_dir_for_model_mem(llm_family, llm_spec)
```

Comment:
Can we make it more clever? If the model is cached locally, we read it from local, otherwise downloading first?

Reply:
As you mention earlier, the symlink of model downloaded from modelscope by xinference-worker is using related path, which is different from the point of view of supervisor. I still cannot resolve the problem

Reply:
OK, Let's do it later.